src/index.js

```diff
@@ -0,0 +1,6 @@
+import 'style!../sass/lightbox.scss';
+import Lightbox from './Lightbox';
+
+export default {
+  init: (options) => new Lightbox(options)
```

Comment:
Consider 
```
init(options) {
  return new Lightbox(options);
}
```

Arrows as methods have weird semantics if they reference `this`, which is a common thing to do in object methods. Non-blocker.